metrics/rouge/README.md

```diff
@@ -42,10 +42,20 @@ At minimum, this metric takes as input a list of predictions and a list of refer
 {'rouge1': 1.0, 'rouge2': 1.0, 'rougeL': 1.0, 'rougeLsum': 1.0}
 ```
 
+It can also deal with lists of references for each predictions:
+```python
+>>> rouge = evaluate.load('rouge')
+>>> predictions = ["hello there", "general kenobi"]
+>>> references = [["hello", "there"], ["general kenobi", "general yoda"]]
+>>> results = rouge.compute(predictions=predictions,
+...                         references=references)
+>>> print(results)
+{'rouge1': 0.8333, 'rouge2': 0.5, 'rougeL': 0.8333, 'rougeLsum': 0.8333}```
+
 ### Inputs
 - **predictions** (`list`): list of predictions to score. Each prediction
         should be a string with tokens separated by spaces.
-- **references** (`list`): list of reference for each prediction. Each
+- **references** (`list` or `list[list]`): list of reference for each prediction or a list of several predictions. Each
```

Comment:
nit: Should this be "a list of several references for this prediction" as opposed to "a list of several predictions"? I assume the change is meant to surface that each prediction can be correctly mapped to several references.

Reply:
Absolutely, thanks for noticing this!